web/app/components/base/app-icon-picker/utils.ts

```diff
@@ -93,6 +93,10 @@ export default async function getCroppedImg(
         resolve(file)
       else
         reject(new Error('Could not create a blob'))
-    }, 'image/jpeg')
+    }, 'image/png')
   })
 }
+
+export function isAnimatedImage(file: { name: string }) {
```

Comment:
I think in most cases `webp` could be a static image, maybe we can use a better way to determain if an uploaded image is animated? 

I asked ChatGPT, and here's the answer maybe we can refer to:

```javascript
function isAnimatedWebP(file) {
  return new Promise((resolve, reject) => {
    const reader = new FileReader();
    reader.onload = function(event) {
      const data = event.target.result;
      const view = new DataView(data);
      
      // Checking for the WebP RIFF header.
      if (view.getUint32(0, false) !== 0x52494646 /* 'RIFF' */ ||
          view.getUint32(8, false) !== 0x57454250 /* 'WEBP' */) {
        reject(new Error("Not a WebP file"));
        return;
      }
      
      // Checking for the presence of the 'ANIM' chunk.
      let offset = 12;
      while (offset < view.byteLength) {
        const chunkType = view.getUint32(offset, false);
        const chunkSize = view.getUint32(offset + 4, true);
        
        // 'ANIM'
        if (chunkType === 0x414E494D /* 'ANIM' */) {
          resolve(true);
          return;
        }
        
        // Move to the next chunk (chunkSize is padded to even number of bytes)
        offset += 8 + chunkSize + (chunkSize % 2);
      }
      resolve(false); // No animation chunk found
    };
    reader.onerror = function() {
      reject(new Error("File reading failed"));
    };
    reader.readAsArrayBuffer(file);
  });
}
```